sql/core/src/test/resources/sql-tests/inputs/postgreSQL/join.sql

```diff
@@ -6,6 +6,11 @@
 -- Test JOIN clauses
 -- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/join.sql
 --
+
+--SET spark.sql.autoBroadcastJoinThreshold=10485760
```

Comment:
Also, this looks like a magic number, `10485760`. Could you explain why you choose this value? I don't see this kind of value from the original `https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/join.sql`.

Reply:
I just copied them from the other join-related tests (e.g., https://github.com/apache/spark/blob/master/sql/core/src/test/resources/sql-tests/inputs/natural-join.sql#L2-L4), so I'm not sure about why the value chosen. I think this configuration is just to prohibit broadcast hash joins for tests.

Reply:
@dongjoon-hyun @maropu 
 `10 * 1024 * 1024 = 10485760` 
This config's default value is `10m`

Reply:
right, it is the default value

Reply:
Thanks, @mgaido91 !